socrata-http-jetty/src/main/scala/com/socrata/http/server/AbstractSocrataServerJetty.scala

```diff
@@ -180,11 +181,9 @@ abstract class AbstractSocrataServerJetty(handler: Handler, options: AbstractSoc
 
   private def gzipHandler(underlying: Handler): Handler = gzipOptions match {
     case Some(opts) =>
-      val gz = new GzipHandler() {
```

Comment:
This was used as a workaround for https://github.com/jetty/jetty.project/issues/5171.

Reply:
Note: This does deprecate the "excluded user agents" option.